src/test/java/com/tngtech/configbuilder/ConfigBuilderIntegrationTest.java

```diff
@@ -20,10 +20,14 @@
 
 public class ConfigBuilderIntegrationTest {
 
-    @Rule
-    public SystemOutRule systemOut = new SystemOutRule();
+    static {
+        Locale.setDefault(Locale.US);
```

Comment:
Should we move this to `testValidation`, which seems to be the only test which relies on it?

And ideally, we'd reset the default `Locale` to its previous value after the test, e.g. using [Junit 5 Pioneer's `@DefaultLocale`](https://wesome.org/junit-5-pioneer-defaultlocale) to avoid influencing other tests (but I don't think that this is an actual issue in this case).

Reply:
I think this was simply a leftover from local testing and can be removed again. It doesn't matter for Github actions, but in a non-English system environment, lots of things can go wrong. Most libraries rely on the default JVM locale, but Hibernate for example seems to check the system's locale setting.

I'll recheck.

Reply:
If your default locale is `GERMAN`, `hasMessageContaining("must not be null")` will indeed fail.